src/tools/miri/tests/pass/track-alloc-1.stderr

```diff
@@ -1,5 +0,0 @@
-note: tracking was triggered
-   |
-   = note: created extern static allocation of SIZE bytes (alignment ALIGN bytes) with id 1
-   = note: (no span available)
-
```

Comment:
I'm a bit baffled why this disappears. The test is actually starting MIR interpretation, and there are no allocations beyond the one with id `1`. So my guess is MIR interpretation doesn't create it, but const prop or some other analysis pass does and we just reuse the cached one.

Reply:
Did AllocIds shift? The PR might evaluate more things earlier now so const-eval uses AllocIds that previously were still available to Miri.

Reply:
oh 🤦 I didn't go up far enough with the ids, because I started seeing other ids that I thought came later.